reference/outlook/preview/SeriesTime.md

```diff
@@ -0,0 +1,391 @@
+	var context = asyncResult.context;
+	var recurrence = asyncResult.value;
+	var endDate = recurrence.seriesTime.getEndDate();
+}
+```
+
+##### Requirements
+
+|Requirement|Value|
+|---|---|
+|[Minimum mailbox requirement set version](../tutorial-api-requirement-sets.md)|Preview|
+|[Minimum permission level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)|ReadItem|
+|Applicable Outlook mode|Compose or read|
+
+#### getEndTime() → {String}
+
+Gets the start time of the meeting/appointment of a recurrence pattern in whichever time zone that the user/add-in set the recurrence pattern in the following ISO 8601 format:
+"THH:mm:ss:mmm"
+
+##### Examples
+
+This example gets the end time of a recurrence.
+
+```js
+Office.context.mailbox.item.recurrence.getAsync(callback);
+function callback(asyncResult){
+	var context = asyncResult.context;
+	var recurrence = asyncResult.value;
+	var endDate = recurrence.seriesTime.getEndTime();
+}
+```
+
+##### Requirements
+
+|Requirement|Value|
+|---|---|
+|[Minimum mailbox requirement set version](../tutorial-api-requirement-sets.md)|Preview|
+|[Minimum permission level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)|ReadItem|
+|Applicable Outlook mode|Compose or read|
+
+#### getStartDate() → {String}
+
+Gets the start date of a recurrence pattern in the following [ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) date format:
+"YYYY-MM-DD"
+
+##### Examples
+
+This example gets the start date of a recurrence.
+
+```js
+Office.context.mailbox.item.recurrence.getAsync(callback);
+function callback(asyncResult){
+	var context = asyncResult.context;
+	var recurrence = asyncResult.value;
+	var endDate = recurrence.seriesTime.getStartDate();
+}
+```
+
+##### Requirements
+
+|Requirement|Value|
+|---|---|
+|[Minimum mailbox requirement set version](../tutorial-api-requirement-sets.md)|Preview|
+|[Minimum permission level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)|ReadItem|
+|Applicable Outlook mode|Compose or read|
+
+#### getStartTime() → {String}
+
+Gets the start time of the meeting/appointment of a recurrence pattern in whichever time zone that the user/add-in set the recurrence pattern in the following ISO 8601 format:
+"THH:mm:ss:mmm"
+
+##### Examples
+
+This example gets the start time of a recurrence.
+
+```js
+Office.context.mailbox.item.recurrence.getAsync(callback);
+function callback(asyncResult){
+	var context = asyncResult.context;
+	var recurrence = asyncResult.value;
+	var endDate = recurrence.seriesTime.getStartTime();
+}
+```
+
+##### Requirements
+
+|Requirement|Value|
+|---|---|
+|[Minimum mailbox requirement set version](../tutorial-api-requirement-sets.md)|Preview|
+|[Minimum permission level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)|ReadItem|
+|Applicable Outlook mode|Compose or read|
+
+#### setDuration(int minutes)
```

Comment:
do we usually put the type of the variable in front of variable name? I think we usually just say the variable name and then list the type in the table beneath